src/parsley/utils.js

```diff
@@ -93,6 +101,22 @@ define('parsley/utils', function () {
         .toLowerCase();
     },
 
+    objectCreate: Object.create || (function() {
```

Comment:
Could benefit a little comment / doc on how this method works and what it does. Not mandatory but a plus :)